andela/pippin-ah-backend
# [Feature:  #162727564] List authors

#### What does this PR do?
- Get the list of authors and their profile

#### Description of Task to be completed?
- [x] add new route to get all authors
- [x] verify token when one navigates to that route
- [x] add controller to get the list and profile of users who have article in the article table

#### How should this be manually tested?
- Check out into the `feat/162727564/list-users` branch
- Ensure there are articles in article table
- Run `npm start dev` and navigate to GET /api/v1/authors
- You will see the list and profile of users who have written article

#### What are the relevant pivotal tracker stories?
[#162727564](https://www.pivotaltracker.com/story/show/162727564)
